src/Aspire.Hosting.PostgreSQL/PostgresContainerImageTags.cs

```diff
@@ -7,10 +7,10 @@ internal static class PostgresContainerImageTags
 {
     public const string Registry = "docker.io";
     public const string Image = "library/postgres";
-    public const string Tag = "16.2";
+    public const string Tag = "16.4";
     public const string PgAdminRegistry = "docker.io";
     public const string PgAdminImage = "dpage/pgadmin4";
-    public const string PgAdminTag = "8.8";
+    public const string PgAdminTag = "8.9";
```

Comment:
I see an 8.11 now https://hub.docker.com/r/dpage/pgadmin4/tags

Reply:
You are right, I missed it because the tool I'm running was listing them alphabetically 😓

![image](https://github.com/user-attachments/assets/a6f64d21-4965-433b-a960-dab02bfd4f46)